lib/groonga/command/logical-range-filter.rb

```diff
@@ -44,6 +44,7 @@ def parameter_names
             :limit,
             :filter,
             :output_columns,
+            :sort_keys,
```

Comment:
This is wrong order. `:use_range_index` and `:post_filter` is missing: https://github.com/groonga/groonga/blob/master/plugins/sharding/logical_range_filter.rb#L17-L18